hoomd/md/AllBondPotentials.h

```diff
@@ -29,21 +31,42 @@ namespace hoomd
 namespace md
     {
 //! Bond potential force compute for harmonic forces
-typedef PotentialBond<EvaluatorBondHarmonic> PotentialBondHarmonic;
+typedef PotentialBond<EvaluatorBondHarmonic, BondData> PotentialBondHarmonic;
 //! Bond potential force compute for FENE forces
-typedef PotentialBond<EvaluatorBondFENE> PotentialBondFENE;
+typedef PotentialBond<EvaluatorBondFENE, BondData> PotentialBondFENE;
 //! Bond potential force compute for Tethering forces
-typedef PotentialBond<EvaluatorBondTether> PotentialBondTether;
+typedef PotentialBond<EvaluatorBondTether, BondData> PotentialBondTether;
+
+//! Mesh Bond potential force compute for harmonic forces
+typedef PotentialBond<EvaluatorBondHarmonic, MeshBondData> PotentialMeshBondHarmonic;
+//! Mesh Bond potential force compute for FENE forces
+typedef PotentialBond<EvaluatorBondFENE, MeshBondData> PotentialMeshBondFENE;
```

Comment:
FENE is typically a potential used for polymer simulations to avoid bond crossings. Do you see a strong need for users of meshes to utilize FENE potentials?